.gitignore

```diff
@@ -115,6 +119,20 @@ venv.bak/
 /**/.idea/*
 .idea/*
 
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Editor directories and files
+.idea
+.vscode
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.sw?
+
```

Comment:
What are these files?

Reply:
These are Visual Studio (not Code) files.